test/cmdlineTests.sh

```diff
@@ -436,15 +436,18 @@ SOLTMPDIR=$(mktemp -d)
     # The contract should be compiled
     if [[ "$result" != 0 ]]
     then
+        printError "Failed to compile a simple contract from standard input"
         exit 1
     fi
 
     # This should not fail
     set +e
-    output=$(echo '' | "$SOLC" --ast - 2>/dev/null)
+    output=$(echo '' | "$SOLC" --ast-json - 2>/dev/null)
+    result=$?
     set -e
-    if [[ $? != 0 ]]
+    if [[ $result != 0 ]]
```

Comment:
There's no `--ast` option (and I think there never was?) but this was not detected becuase `$?` in the `if` is the result of the `set` command, not the compiler.

I suspect this was supposed to be either `--ast-compact-json` or `--ast-json`. Or maybe `--combined-json ast`?